src/lib/regions.ts

```diff
@@ -0,0 +1,49 @@
+import Client from '../client'
+
+interface RegionResponse {
+  name: string
+  code: string
+  latitude: number | null
+  longitude: number | null
+  gatewayAvailable: boolean
+  requiresPaidPlan: boolean
+}
+
+interface PlatformResponse {
+  requestRegion: string | null
```

Comment:
```suggestion
  requestRegion: string
```

ditto, seems like the field is always defined https://github.com/superfly/flyctl/blob/master/api/types.go#L47

Reply:
<img width="1656" alt="Screenshot 2023-12-20 at 9 31 10 PM" src="https://github.com/supabase/fly-admin/assets/51415676/0e84163b-1f81-4680-8719-97196c12ce17">

I was going off the playground and not the code, you are right